lib/tlIO/FFmpegReadVideo.cpp

```diff
@@ -250,6 +244,43 @@ namespace tl
                         _info.pixelType = image::PixelType::YUV_444P_U16;
                     }
                     break;
+                case AV_PIX_FMT_YUVA420P:
+                case AV_PIX_FMT_YUVA422P:
+                case AV_PIX_FMT_YUVA444P:
+                    //! \todo Support these formats natively.
+                    // if (options.yuvToRGBConversion)
+                    // {
+                    _avOutputPixelFormat = AV_PIX_FMT_RGBA;
+                    _info.pixelType = image::PixelType::RGBA_U8;
+                    // }
+                    // else
+                    // {
+                    //     //! \todo Use the _info.layout.endian field instead of
+                    //     //! converting endianness.
+                    //     _avOutputPixelFormat = AV_PIX_FMT_YUV444P16LE;
+                    //     _info.pixelType = image::PixelType::YUV_444P_U16;
+                    // }
+                    break;
+                case AV_PIX_FMT_YUVA444P10BE:
+                case AV_PIX_FMT_YUVA444P10LE:
+                case AV_PIX_FMT_YUVA444P12BE:
+                case AV_PIX_FMT_YUVA444P12LE:
+                case AV_PIX_FMT_YUVA444P16BE:
+                case AV_PIX_FMT_YUVA444P16LE:
+                    //! \todo Support these formats natively.
+                    // if (options.yuvToRGBConversion)
+                    // {
+                        _avOutputPixelFormat = AV_PIX_FMT_RGBA64LE;
```

Comment:
Can this be `AV_PIX_FMT_RGBA64` instead?

Reply:
Sure thing.  I updated the PR.